db/migrate/20210915095847_create_monthly_statistics_report.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMonthlyStatisticsReport < ActiveRecord::Migration[6.1]
+  def change
+    create_table :monthly_statistics_reports do |t|
+      t.timestamps
+      t.json :statistics
```

Comment:
`jsonb` does not respect key order :-(

Reply:
It may not matter but `jsonb` might give us better options to trawl the raw data (in ways we've not yet thought of).

Do you think it's worth maintaining a hard-coded array of keys for each report that defines the content and order of each report type? It would make it easy to make certain changes in the future like changing the order of columns.

Reply:
Sounds like a good idea @stevehook 
What do you think @duncanjbrown?

Reply:
One thing I quite like about the current approach is that the components can be really dumb because they just deal in hashes — this would make everything slightly more complicated. Given the low volume of of reports (once a month) and the fact that the queries are already defined in code, it doesn't seem very likely that we'll need to query the reports themselves? (Or that not having JSONB would be much of an impediment if it turned out we did need to, as we could work quite easily with small amounts of plain JSON in Ruby).